va/va_vpp.h

```diff
@@ -303,10 +306,77 @@ typedef enum _VAProcColorStandardType {
     VAProcColorStandardSMPTE240M,
     /** \brief Generic film. */
     VAProcColorStandardGenericFilm,
+    /** \brief sRGB. */
+    VAProcColorStandardSRGB,
+    /** \brief stRGB. */
+    VAProcColorStandardSTRGB,
+    /** \brief xvYCC601. */
+    VAProcColorStandardXVYCC601,
+    /** \brief xvYCC709. */
+    VAProcColorStandardXVYCC709,
+    /** \brief ITU-R BT.2020. */
+    VAProcColorStandardBT2020,
```

Comment:
Can you specify exactly what each of these mean?  (See the changes I added to the other values of this enum in #127 .)

In particular, I'm wondering what the BT2020 case actually is here - e.g. is it BT.2020 colour space, with the usual SDR transfer characteristic, and using BT.2020 non-constant luminance matrix coefficeints?

Reply:
yes, will refer #127